docs/en/stack/monitoring/esms.asciidoc

```diff
@@ -0,0 +1,208 @@
+[role="xpack"]
+[[esms]]
+== {esms}
+
+The {esms} ({esms-init}) is a monitoring cluster on {ecloud}. Elastic provides
+and maintains {esms-init} for self-managed commercial customers. If you send
+your monitoring data to {esms-init}, it can also be used by Elastic support to
+provide better and faster incident resolution.
+
+NOTE: You must obtain your {esms-init} cluster URLs and credentials from the
+Elastic support team.
+
+[discrete]
+[[esms-elasticsearch]]
+=== Collecting monitoring data about {es}
+
+There are two methods for collecting and sending data about the health of your
+production cluster to {esms-init}: {metricbeat} or collectors and exporters.
+
+TIP: If you want to monitor {ls}, you must use collectors and exporters to route
+data from the production cluster to {esms-init}. Otherwise, it is simplest to
+use {metricbeat}. 
+
+To use {metricbeat}:
+
+. If your production cluster is stopped,
+{ref}/starting-elasticsearch.html[start it].
+
+. Configure the following settings on each node in the production cluster:
++
+--
+[source,js]
+----------------------------------
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true,
+    "xpack.monitoring.elasticsearch.collection.enabled": false
+  }
+}
+----------------------------------
+// CONSOLE
+
+Alternatively, you can put those settings in the `elasticsearch.yml` file on
+each node. This method, however, requires you to restart each node.
+
+The `xpack.monitoring.enabled` setting must also have a value of `true`, which
+is its default value. If you have set it to `false` in the `elasticsearch.yml`
+file on any node, you must change or remove it.
+
+For more information about these settings, see
+{ref}/monitoring-settings.html[Monitoring settings in {es}].
+--
+
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on each
+node.
+
+. Optional: Disable the system module in {metricbeat}. +
```

Comment:
Should we explain why we include this step, especially since it's marked as optional. That way the user can decide whether they should take it or not.

To recap, the reason for this step is to stop Metricbeat from indexing metrics that are currently not used by the ESMS cluster.

Reply:
Thanks, I've added explanatory text in https://github.com/elastic/elasticsearch/pull/42601 and https://github.com/elastic/kibana/pull/37198 and pulled that information into this page too.